khronicle-test/build.gradle.kts

```diff
@@ -0,0 +1,32 @@
+plugins {
+    kotlin("multiplatform")
+    id("kotlinx-atomicfu")
+    id("org.jmailen.kotlinter")
+    jacoco
+    id("org.jetbrains.dokka")
+    id("com.vanniktech.maven.publish")
```

Comment:
Not sure how restrictive you wanted to be in its usage, but you could "hide" this module from consumers by not publishing it. 🤷 

```suggestion
```

Reply:
This needs to be published because I want to use it downstream -- my hope with the second module was simply that nobody would add an `implementation(libs.khronicle.test)` to their `main` source set.